### ISSUE TITLE
Error message does not reflect field names

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#4510]
Version - v1.0.1

Steps to Reproduce: 
1.  Go to Active Directory or LDAP
2.  Do not fill out any fields except Username & Password
3.  Click Authenticate

Results: "Domain" is required & "Server" is required messages do not reflect the UI's field names so it can be confusing.

Expected:  Make the error message more user friendly

**Comment by vincent99** [comment]
If you want to do this, move all the `label`, `prompt`, and `placeholder` translations (so they stay together instead of having a whole separate tree for the prompt or placeholder) for each field into  the `model` top level key.

e.g.

```
model:
  container:
    image:
      label: Image
      placeholder: "e.g. ubuntu:xenial"
```

Then update the references to the old keys, and update `app/utils/cattle-transitioning-resource`'s `validationErrors()` to look for the `.label` key first and use that if found.

**Comment by tfiduccia** [comment]
Version - 1.6 master 10/16
Not fixed
LDAP still says "Domain" and "Server"
![image](https://user-images.githubusercontent.com/11514927/31639802-9571b8ba-b28f-11e7-90cc-7c762782283c.png)
Domain = Hostname or IP Address
Server = User Search Base

**Comment by loganhz** [comment]
The merged PR is for 2.0.
Submitted a new PR for 1.6.
Thanks for your test!

**Comment by tfiduccia** [comment]
version - 1.6.11-rc6
Verified fixed.  Local LDAP is fixed.
